src/master-playlist-controller.js

```diff
@@ -931,6 +955,17 @@ export class MasterPlaylistController extends videojs.EventTarget {
     }
   }
 
+  pausePlaylistLoading() {
```

Comment:
Added a function to pause loaders. I wonder if this should just be a part of `pauseLoading`?

Reply:
I think it should. If it is, then we can remove the error listener on MPC and rely on the tech error listener in `videojs-http-streaming.js`.